rebar.lock

```diff
@@ -1,12 +1,17 @@
-{"1.1.0",
+{"1.2.0",
```

Comment:
do we really want to bump rebar3 version? lots of our repos still use 1.1.0 and we will start seeing lots of warnings